# The entanglement of 3qubits / GHZ state
Greenberger–Horne–Zeilinger state is the state which as entanglement of 3qubits. Let's see the circuit.

In [0]:
!pip install blueqat

In [2]:
from blueqat import Circuit                                                            

Circuit().h[:2].x[2].cx[1,2].cx[0,2].h[:].m[:].run(shots=100)

Counter({'000': 53, '111': 47})

We have half of '000' and another half of ''111'.